reference-architecture/rhv-ansible/inventory/hypervisors/group_vars/hypervisors

```diff
@@ -0,0 +1 @@
+ansible_ssh_user: root
```

Comment:
Maybe place this in ansible.cfg if you can...for example

remote_user = ec2-user